src/pages/authenticate/credentialScopes.js

```diff
@@ -0,0 +1,14 @@
+import { UserIcon, CheckIcon } from '@nomios/web-uikit';
```

Comment:
Question: should this go to the [data](https://github.com/ipfs-shipyard/nomios-web/tree/master/src/shared/data) folder? No strong opinion, just asking.

Reply:
No strong opinion as well. Moved it already to `data` folder.